### ISSUE TITLE
frigate scuttle damage

### ISSUE THREAD

**Issue by Novaras** [HW-PlayersPatch/2.3#36]
Frigates probably shouldn't deal any scuttle damage at all, this is _never_ a tactical sacrifice being made or some other ass-pull logic.

Even worse is that this mechanic is naturally fine for some units (most corvettes, frigates w/e) and *unreliably terrible* for others such as bombers.

**Comment by Echo1707** [comment]
I agree. Should that be applied to destroyers or bc/hc too?

On a side note, weren't strikecrafts supposed to clear the blast zone automatically?

**Comment by Novaras** [comment]
lower it? its already so low that you can occasionally be sitting right next to a frigate and take no damage, or occasionally you'll get oneshot through no action of either player, this mechanic is garbage at best even in other scenarios, but its just criminal here

**Comment by cloaked2222** [comment]
Explosion damage add realism to the game. Even fighters/corvettes have it. It is not to be removed, but rather tuned properly. 

There is a regular death, and a scuttle death. The later is equivalent to a reactor going critical and does more damage over a greater range. You can also tune how long it takes a ship to die.

Tune away.

**Comment by Echo1707** [comment]
**TL;DR**: What do you guys think of flat-out halving frigates death and scuttle damage? Or should it be more like "reducing the damage until bombers stay alive beyond this range for frigate death and beyond this other range for scuttle"?
___
Just few days ago I had a match using the Community Patch with 2 other random players. I took an HW1 race and opened with light corvettes. Me and one of these players both expanded towards each other. He was Vaygr and he was making ints, so I rushed him while I was quickly researching ints on my own. I killed a few of his squadrons (some he retreated back to his MS), then I opted for bombers to kill his carrier's engine and secure the kill. When his carrier was about to die, he scuttled it, **killing all my swarm**. Next thing I know is that he has 1-2 new carriers back at his MS and he was building a swarm.

Now, let's imagine what would have happened if that match wasn't a FFA but an 1vs1. He would have swarmed me back and I would have scuttled my carrier when it was about to die, killing his own swarm in return :) But guess what? I would have been left with only my MS because HW1 carriers take ages to unlock and build, while he had 2 carriers. GG.

Was losing my swarm a punishment because I researched ints instead of frigates and didn't went after his MS collectors with my light corvettes? As Vaygr, the best counter to light corvettes are missile corvettes (I had a swarm of lights while he was making assault crafts, so he couldn't have built max numbers and would have suffered some casualties anyways because lances are still fighters). So it seems like assaults would have been the best choice after unlocking light vettes... but 1) I really wanted to kill that carrier, and 2) back at his MS he was making only assault crafts! No lances nor missile corvettes!

So, why was he punished for _not_ countering my corvettes by losing just a few AC squads, a few collectors and a carrier, while I was punished for killing his carrier and not going after his MS collectors by losing all my swarm thanks to him pushing a button?

I repeat that I was playing with Community Patch and not 2.3, I don't know if Artillery increased scuttle/death damage/range over vanilla, but if my swarm is all over a scuttling carrier, what do I do? Do I tell all my swarm to move in one direction? It's quick, but very few of my units would actually move facing away from the carrier and some of them would actually move _toward_ the carrier. Should I split my units into two groups and having them move in opposite directions? Sounds better even if a little more micro is involved, but maybe it requires more effort than pushing a button to scuttle...

So HW1 carriers should have a greater scuttle damage than HW2 carriers to compensate their slower build time. But what about standard death damage? Simple: that can't be higher. If I leave my carrier over a RU patch and it gets destroyed by frigates/dds/hcs/bcs, I'd be damaging my own nearby units (like collectors and support frigates), but it's already a huge blow for a HW1 player to have his carrier destroyed, I don't understand the need to rub salt into it.
On the other way around: it's not written anywhere in the tooltip that carriers are good vs frigates, and in fact they are not; but it's also not written anywhere that carriers can be good against swarms if scuttled at the right time. I'd rather see PD guns being improved against fighters and corvettes than killing them via explosions.

Speaking of which, what is the role of HW1 bombers actually? They aren't very good against subsystems, so you need more of them to do the same job, and this translates in more RU used and less available space for ints. It seems to me that their primary job is to kill frigates and other capships. Bombers are already weak enough against fighters and corvettes that they die faster than they kill frigates, let alone a tanky HW2 DD or a BC which has PDs. So if there are a ton of anti-corvette and anti-frigates (but nothing anti-fighters) frigates coming my way, I make bombers to counter them. If my bombers where to automatically avoid blast zones I would have nothing to say, but they don't, if their fly-by gets them near an exploding vessel they are fine with dieing with it.

This post is becoming huge and I'm repeating myself, and since I'm not a native english speaker I know that I might sound too direct/harsh/sarcastic than I intend. So to conclude this, and downplaying a bit, I'd like to show you two videos to laugh at (at least the second... the first makes me cry to this day actually):
1) [Homeword Remastered players view 1v1 (Xenofix vs Echo7)](https://www.youtube.com/watch?v=HlpzvVwwFp8) by Xenofix: ~~this video shows how pathetic my knowledge of the game is, so don't take too seriously what I'm saying~~ It's okay that HW2 motherships don't deal too much damage while exploding, because if there are still carriers alive they can build shipyards and still fight back (and win, _sob_). HW1 factions don't have this luxury, so... if a Vaygr carrier is able to destroy an entire swarm (again, it happened on Community Patch, but it could happen in 2.3 too), does this mean that a HW1 carrier should destroy all DDs and BCs around the map? Because a HW1 faction can't make them anymore, but it should still have the opportunity to win like Xenofix had ^^ :P (I'm obviously joking here, I don't intend it seriously... because you know... it would kill my HW1 carrier too :P )
2) [Homeworld Do not disturb the hornets nest!](https://www.youtube.com/watch?v=HEQE73RRtz8) by endoskeptic: my greatest argument against ships death damage! :')

**Comment by cloaked2222** [comment]
Your english is quite good actually. :)

ArtilleryWhore started his CP mod in 2015 based on HWR v1.0, and basically did a ton of work on his own to completely re-balance the game.

2.3 started based on HWR v2.205.

I'm sure balance is VERY different. The CP patch could have 10x carrier skuttle damage for all I know. You'd have to check the files.

In 2.3, I don't think I've seen a carrier skuttle ever do much in a skilled game. I know I've seen ghost try and fail many times tho, lol.

**Comment by Novaras** [comment]
Did we ever find out what causes some ships to take higher damage from scuttles? HW1 frigates are the best example of this.

If you let ships take longer to die, they can have enough time for death volleys no? I would cull frigate scuttle so it at *most* does 50% to fighters. I honestly don't think anyone cares about or often knows about scuttle damage on ships smaller than destroyers, so I think we can be reasonably liberal with it...

**Comment by cloaked2222** [comment]
HW1 frigates taking more scuttle damage is still a mystery (i did take a decent stab at it). It may be possible to figure out the problem by swapping .ship and .hod files etc between Hiig and Kus, until you find the source of the problem. I suspect however that we may not be able to fix this problem.

By taking longer to die, i mean their death animation. Ships do not fire during their death animation. Its like watching a Hiig MS die, it tilts off center and then starts to explode. Its just an animation at this point until the big bang.

Echo, I'd run some tests like 30 bombers vs 6 ions frigs and 30 ints vs 6 ion frigs, to see how frigate death explosions actually affect fighters. Then I'd tweak frigate blast damage as necessary so fighters aren't affected too much. You've got lots of properties to tweak as i explained above. If you end up with say a 50+% nerf or whatever, i'm sure that's fine.

**Comment by cloaked2222** [comment]
Pie in the sky
